specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure.Management/tspconfig.yaml

```diff
@@ -33,3 +33,7 @@ options:
     clear-output-folder: true
     namespace: Microsoft.DevOpsInfrastructure
     model-namespace: false
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    clear-output-folder: true
+    namespace: "Azure.ResourceManager.DevOpsInfrastructure"
```

Comment:
missing `package-dir`.
SDK generation is failing: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4053329&view=logs&j=03afb3bb-7296-55ad-aa07-ceee610c73b2&t=0f320c29-1c90-5304-d2d3-c58a6bb76437&l=198

Reply:
fixed